src/ballet/bn254/fd_bn254.c

```diff
@@ -166,7 +166,9 @@ fd_bn254_g1_add_syscall( uchar       out[64],
     return -1;
   }
   uchar FD_ALIGNED buf[128] = { 0 };
-  fd_memcpy( buf, in, in_sz );
+  if( in_sz ) {
+    fd_memcpy( buf, in, in_sz );
+  }
```

Comment:
@0x0ece Sorry, would you mind reverting this change? 
Unlike stdlib memcpy, fd_memcpy is actually fine with a 0 size argument for all supported build targets (test_util_base verifies that). The reason is that 0 sz stdlib memcpy is fine on glibc, GCC, and Clang.

There are still some "insane" targets like CBMC or UBSan that complain on 0 sz stdlib memcpy, but for those, fd_memcpy does this branch under the hood.

Reply:
@ripatel-fd I've got this issue with:
```
MACHINE=linux_clang_x86_64 EXTRAS="asan ubsan rpath" make run-test-vectors
```